src/components/templates/Jazzbar/components/JazzBarTableComponent/JazzBarTableComponent.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
```

Comment:
This file seems to duplicate our generic `src/components/molecules/TableComponent/TableComponent.tsx`

What's the reasoning/need for duplicating it?

Reply:
The only differences appear to be:

```diff
diff --git a/src/components/molecules/TableComponent/TableComponent.tsx b/src/components/templates/Jazzbar/components/JazzBarTableComponent/JazzBarTableComponent.tsx
index 7c639bca..48aaef51 100644
--- a/src/components/molecules/TableComponent/TableComponent.tsx
+++ b/src/components/templates/Jazzbar/components/JazzBarTableComponent/JazzBarTableComponent.tsx
@@ -1,10 +1,15 @@
 import React from "react";
 import { TableComponentPropsType } from "types/Table";
-import "./TableComponent.scss";
 import { DEFAULT_PARTY_NAME, DEFAULT_PROFILE_IMAGE } from "settings";
 import { useSelector } from "hooks/useSelector";
 import { currentVenueSelectorData } from "utils/selectors";
 
+import "./JazzBarTableComponent.scss";
+
+// @debt THIS COMPONENT IS THE COPY OF components/molecules/TableComponent
+// The reason to copy this was the lack of time to refactor the whole thing, so the
+// safest approch (not to break other Venues that rely on TableComponent) is to copy this component
+// It needs to get deleted in the future
 const TableComponent: React.FunctionComponent<TableComponentPropsType> = ({
   users,
   onJoinClicked,
@@ -24,14 +29,8 @@ const TableComponent: React.FunctionComponent<TableComponentPropsType> = ({
     table.capacity && table.capacity - usersSeatedAtTable.length;
   const full = numberOfSeatsLeft === 0;
   return (
-    <div className={`table-component-container ${table.reference}`}>
-      <div
-        className="table-item"
-        style={{
-          height: `${table.rows && table.rows * 50 + 65}px`,
-          width: `${table.columns && (table.columns + 1) * 55}px`,
-        }}
-      >
+    <div className={`jazzbar-table-component-container ${table.reference}`}>
+      <div className="table-item">
         <div className="table-occupancy-information red-text">
           {locked ? "locked" : full ? "full" : ""}
         </div>
```